cmd/storagenode/alpha/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -euo pipefail
+
+if [[ ! -f "config/config.yaml" ]]; then
+	./storagenode setup --config-dir config --identity-dir identity
+fi
+
+RUN_PARAMS="${RUN_PARAMS:-} --config-dir config"
+RUN_PARAMS="${RUN_PARAMS:-} --identity-dir identity"
+
+RUN_PARAMS="${RUN_PARAMS:-} --kademlia.bootstrap-addr=bootstrap.storj.io:8888"
+RUN_PARAMS="${RUN_PARAMS:-} --metrics.app-suffix=-alpha"
```

Comment:
It's not using `STORJ_METRICS_APP_SUFFIX` env variable

Reply:
The only flags we should pass to `storagenode run` should be just `--config-dir` and `identity-dir`. All the rest should be configured via the standard Corba env vars, i.e. `STORJ_KADEMLIA_EXTERNAL_ADDRESS` instead of just `ADDRESS`.